packages/auth/src/grant/routes.ts

```diff
@@ -101,7 +102,12 @@ async function createGrant(
   try {
     noInteractionRequired = canSkipInteraction(deps.config, ctx.request.body)
   } catch (err) {
-    ctx.throw(400, 'identifier_required', { error: 'identifier_required' })
+    ctx.throw(400, GNAPErrorCode.InvalidRequest, {
+      error: {
+        code: GNAPErrorCode.InvalidRequest,
+        description: 'access identifier required'
+      }
```

Comment:
Should we call `generateGNAPErrorResponse` here?

Reply:
Good catch, yes we should